basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    public static int[] roll(int rollTimes) {
+        int[] rollResultArray = new int[rollTimes];
+        for (int i = 0; i < rollTimes; i++){
+            int rollResult = (int)((Math.random() * (6 - 1)) + 1);
+            rollResultArray[i] = rollResult;
+        }
+        return rollResultArray;
+    }
+
+    public static boolean containsDuplicates(int[] numArray) {
+        boolean result = true;
+        for (int i = 0; i < numArray.length; i++) {
+            for (int j = 0; j < numArray.length; j++){
+                if(numArray[i] == numArray[j]) {
+                    result = true;
+                } else result = false;
```

Comment:
Since we reset whether there are duplicates or not at each index, even if we find a duplicate along the way, if we later see something that isn't a duplicate, it'll be reset.